### DISCUSSION TITLE
add autoupdate

### DISCUSSION THREAD

**Discussion by ThemisterNaim** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#56] [Ideas]
maybe teh app can make once a day an search for update

**Comment by HerMajestyDrMona** [comment]
It's against the core "guidelines" of this program: https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/blob/main/README.md

> Windows11DragAndDropToTaskbarFix has no auto-update function and never connects to the Internet, so you don't need to worry about your privacy.

When a program connects to the Internet it's way more often considered as a risky choice than when no network functions are included at all.

The updates are not released that often anymore, and hopefully in few months Microsoft releases a new Windows 11 build with the drag and drop working. So I would say it's better to keep it as simple and portable as possible.
